test/ui-pytest/gui/elements/qt/object.py

```diff
@@ -117,6 +117,11 @@ def _hover():
 
         assert driver.waitFor(lambda: _hover(), timeout_msec)
 
+    @allure.step('Hover and return')
```

Comment:
If you need hover - use hover that we have If you need to return self in hover - add it, we don't need the method that executes the same things

Reply:
Fixed